core/src/client/client.rs

```diff
@@ -648,6 +648,9 @@ impl BlockChainClient for Client {
         self.importer.miner.count_pending_transactions(range)
     }
 
+    fn future_ready_transactions(&self, range: Range<u64>) -> PendingSignedTransactions {
+        self.importer.miner.future_ready_transactions(range)
+    }
```

Comment:
Please add a new line here.
Other functions in this file have spaces in this position.